### ISSUE TITLE
Find project collaborator(s)

### ISSUE THREAD

**Issue by animalize** [animalize/pyzstd#21]
Find project collaborator(s) to release new versions in my absence.
About 2~3 versions of `zstd` are released every year, and a new version of `pyzstd` needs to be released at this time.

I recently changed the status of the project from **Beta** to **Stable**. 
As I said in https://github.com/animalize/pyzstd/pull/3#issuecomment-825365829, there is basically no need for other maintenance work: 
> I used to spend time checking such details, and manually triggering exceptions to see if them can be handled correctly.
> Once the development of pyzstd module is completed, almost no maintenance is needed.
Basically just update the zstd source code, and use new API in major version updates.

Other precautions have been written in [tech memo](https://github.com/animalize/pyzstd/issues/1). 
If you are interested, I can explain more, such as what I have tried. 

Please ensure that:

1. Long-term service, so it is better for enterprise interested in this.
2. Only build and upload wheels via CI, to prevent the host from being controlled by hacker.

**Comment by animalize** [comment]
If no collaborator(s) are found, and I am unable to appear for a long time (5 months) after `zstd` is upgraded, resulting in the inability to release a new version of `pyzstd`, the person who intends to take over this project seriously can ask the PyPI administrator to transfer `pyzstd` project.
